monosize.config.mjs

```diff
@@ -0,0 +1,12 @@
+// @ts-check
```

Comment:
will this work ? we dont use esm aware TS version. Can we use CJS untill we migrate beyond TS 4.7?

Reply:
Our repo is on 4.7 now (thank you 😉 ). What should I check there?